packages/experiments/src/components/Toggle/Toggle.state.ts

```diff
@@ -32,12 +36,15 @@ export class ToggleState extends BaseState<IToggleProps, IToggleViewProps, ITogg
   };
 
   private _onClick = (ev: React.MouseEvent<HTMLElement>) => {
-    const { disabled, onChange } = this.props;
+    const { label, disabled, onChange } = this.props;
     const { checked } = this.state;
 
     if (!disabled) {
+      let ariaLabel: string = typeof label === 'string' ? label : 'No label';
+      ariaLabel += !checked ? ': On' : ': Off';
```

Comment:
DRY, but I think both of these cases can just be covered by `transformViewProps`.

Reply:
+1

Also, would this be easier to read as a single string interpolation? Has the benefit of being able to be a `const` as well: eg.

    const ariaLabel = `${typeof label === 'string' ? label : 'No label'} ${!checked ? ': On' : ': Off'}`;

Reply:
@JasonGore I'll update the PR with these suggestions. Thanks.

@jdhuntington I was going to try this. However, a scenario came where it became apparent that we have to check that both the `label` and the `onText`/`offText` is present before adding the `:`. This added logic would make the single string interpolation span more than one line. Is it still the preferred approach? For now, I'll update it like that, and we can change it afterwards.